03_letters/brvishnu92/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	x := make(map[rune]int)
+	for _, char := range s {
```

Comment:
FYI: Go uses the term "rune" instead of "char" to avoid the issue of people thinking it might be an 8-bit value. No need to change this now, but a better name would have been just `r` (for rune). `for _, r := range s` is an obvious Go expression (iterating the runes in a string)